### PR TITLE
OBMIN-104 Change CSS to Style-components (SomethingBad)

### PR DESCRIPTION
**Pull request by smilaya**
Change CSS post processor to styled-component for file Something Bad/ unite styles for 2 ErrorPages in styles.js

### CODE REVIEW

**Review by cloudmenson — APPROVED** [review]

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    height: calc(100vh - 3px);
+  }
+`;
+
+export const WrapOImg = styled(WrapImg)`
+  left: 360px;
+`;
+export const WrapGImg = styled(WrapImg)`
+  left: 0;
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:57]: same

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+export const WrapRImg = styled(WrapImg)`
+  right: 50px;
+`;
+
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:61]: same

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    background-size: 300%;
+    background-position: top;
+    background-position-y: 15%;
+  }
+`;
+
+export const WrapOrange = styled(WrapImg)`
+  left: 360px;
+  z-index: -1;
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:94]: same and space

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+export const WrapGreen = styled(WrapImg)`
+  left: 0;
+  z-index: -1;
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:98]: same and space

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+export const WrapRight = styled(WrapImg)`
+  right: 50px;
+  z-index: -1;
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:102]: same

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+export const WrapCenter = styled.div`
+  position: relative;
+  width: 596px;
+  margin: 278px auto 73px;
+`;
+
+export const WrapImg404 = styled(WrapCenter)`
+  position: relative;
+  z-index: 10;
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:13]: export const WrapImg404 = styled.div`
 ${WrapCenter};
  position: relative;
  z-index: 10;
`;

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 7]
```diff
@@ -0,0 +1,186 @@
+import styled, { css } from 'styled-components';
+import SomethingBad from 'assets/img/errorPage/SomethingBad.png';
+
+export const WrapCenter = styled.div`
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:4]: const WrapCenter = css``

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+  z-index: 0;
+  img {
+    max-width: 100%;
+    height: calc(100vh - 3px);
+  }
+`;
+
+export const WrapOImg = styled(WrapImg)`
+  left: 360px;
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:54]: same and space

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+
+export const WrapperLight = styled(WrapperShadow)`
+  width: 483px;
+  position: relative;
+`;
+
+export const WrapperDark = styled(WrapperShadow)`
+  width: 178px;
+  position: absolute;
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:41]: same and space

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+
+  width: 483px;
+  background-color: #e2e2e2;
+  z-index: 0;
+
+  img {
+    display: inline-flex;
+  }
+`;
+
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:32]: same

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+  align-items: center;
+  justify-content: center;
+
+  width: 483px;
+  background-color: #e2e2e2;
+  z-index: 0;
+
+  img {
+    display: inline-flex;
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:30]: You can to create new style component

export const Image  = styled.img`
display: inline-flex;
`

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 12]
```diff
@@ -0,0 +1,186 @@
...
+`;
+
+export const WrapperLight = styled(WrapperShadow)`
+  width: 483px;
+  position: relative;
+`;
+
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:37]: same

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 13]
```diff
@@ -0,0 +1,186 @@
...
+  position: absolute;
+`;
+export const WrapImg = styled.div`
+  position: absolute;
+  top: 0;
+  z-index: 0;
+  img {
+    max-width: 100%;
+    height: calc(100vh - 3px);
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:49]: You can to create a new styled component Img

File: src/main/resources/react/src/pages/ErrorPages/styles.js [hunk 14]
```diff
@@ -0,0 +1,186 @@
...
+
+export const WrapTittleBad = styled.div`
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 10%;
+`;
+
+export const TittleBad = styled(Tittle)`
+  padding: 10px 5px;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ErrorPages/styles.js:112]: same

**Review by Wolshebnik — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**